src/app/components/media/MediaComponent.js

```diff
@@ -143,94 +212,175 @@ class MediaComponent extends Component {
     media.quote = media.media.quote;
     media.embed_path = media.media.embed_path;
 
+    const {
+      fragment,
+      duration,
+      playerRect,
+      playing,
+      progress,
+      start, end, gaps,
+      scrubTo,
+      seekTo,
+      showVideoAnno,
+      time,
+    } = this.state;
+
+    const { currentUser } = this.getContext();
+
+    // console.group("MediaComponent.js");
+    // console.log(this.state);
+    // console.groupEnd();
+
     return (
-      <React.Fragment>
+      <LoadScript
```

Comment:
This forces _all users_ to load Google Maps, for _all media_. And it won't show anything until Google Maps is loaded. That's a problem.

First, please remove `<LoadScript>` from this component, and place it lower down in the hierarchy -- perhaps in `<VideoTimeline>` or even `check-ui`.

After that, if flipping between items causes `<LoadScript>` to warn or error, that'll be a bug in `<LoadScript>`. Try `<LoadScriptNext>` or `useLoadScript`: they use a global variable to track `window.google`.

Reply:
This LoadScript is needed for the timeline and the map in the metadata tab, so it has to be high enough to wrap them both

Reply:
`@react-google-maps/api` is open source too, why would you have to maintain a copy of that? is this core Check product?

Reply:
@Laurian `@react-google-maps/api` has three copies of `<LoadScript>`, and none of them solve the problems I've described in this thread.

> why would you have to maintain a copy of that?

"Maintain" is a leading verb, and it's beside the point. I'm saying we need to _implement_ a fourth copy of `<LoadScript>`. I've described the why and the how in this thread. If we can avoid _maintaining_ our implementation by contributing it upstream, that's a great idea.